src/auth/api/http_middlewares/accesslog.go

```diff
@@ -1,4 +1,4 @@
-package middlewares
+package http_middlewares
```

Comment:
MINOR: Why did you decide on renaming this? IMHO we could either keep `middlewares` or rename it `http` but composed names are not very common in Go as far as I expericed

Reply:
resolved